json-lint/schemas/metrics/providers/custom.json

```diff
@@ -16,9 +16,16 @@
                     },
                     "params": {
                         "type": "object"
+                    },
+                    "platforms": {
```

Comment:
something like in inventory/Adservers/types

"platforms" : {  "$ref": "./types/platformAmpTouch.json" }